Looking at `tomcat/logs/catalina.2022-02-22.log` I am getting an error `Caused by: java.lang.ClassNotFoundException: org.json.JSONException` and basic.sh is returning 404. This is happening in the docker image.
You seem to have taken away org.json, could that be the reason?

--- a/build.xml
+++ b/build.xml
@@ -54,7 +54,7 @@
 
 	<target name="compile" depends="init" description="Compiles all the source files">
 		<!--<javac srcdir="${src}" destdir="${build}" classpath="${include}" /> -->
-        	<javac srcdir="${src}" destdir="${build}">
+        	<javac srcdir="${src}" destdir="${build}" source="1.8" target="1.8">
                   <classpath>
                     <path refid="mvn.compile.classpath" />
                     <fileset refid="include" />

--- a/src/is/ru/cs/nlp/icenlp/web/IceNLPServlet.java
+++ b/src/is/ru/cs/nlp/icenlp/web/IceNLPServlet.java
@@ -259,7 +259,7 @@ public class IceNLPServlet extends HttpServlet
 		TextsResponse textsResponse = analyse(query, english, sentLine, markUnknown, functions, phraseLine, mergeLabels, featureAgreement, showErrors, modelType, showTokenization, strictTokenization, inputTokenizeType, showLemma);
 
 		response.setContentType("application/json");
-		mapper.writeValue(response.getOutputStream(), textsResponse);
+		mapper.writeValue(response.getOutputStream(), textsResponse.asMessage());
 	} catch (Exception e) {
 		e.printStackTrace();
 		response.setContentType("application/json");

